spot-client/setupTests.js

```diff
@@ -6,3 +7,7 @@ Enzyme.configure({ adapter: new EnzymeAdapter() });
 jest.mock('./src/common/logger');
 
 global.fetch = require('jest-fetch-mock');
+
+global.crypto = {
+    getRandomValues: buffer => crypto.randomFillSync(buffer)
```

Comment:
why this change ?

Reply:
just not sure how is it related to adding `request-id` header

Reply:
guid creation. I could mock out the guid method itself, if that's preferred.